### ISSUE TITLE
Instant Crash on startup

### ISSUE THREAD

**Issue by ItsJustDance** [SlimeKnights/Mantle#183]
Version 1.9.27
Window doesn't even open, gives exit code 1. Might be conflict with other mod, just figured I'd send the report so there'd be more info.
Running on modified Create Flavored modpack 
[2022-06-13-7.log.gz](https://github.com/SlimeKnights/Mantle/files/8895290/2022-06-13-7.log.gz)

**Comment by KnightMiner** [comment]
Give me a crash report or I cannot do anything. That log you linked has no crash information.

Also, don't use optifine, that mod breaks more than it fixes.

**Comment by ItsJustDance** [comment]
it didn't create a crash report, and when booted to launcher it never gave the "View Crash Report" option. This is as much info that I have. I'm not dead set on fixing this, but I figured since that version is in beta that I should report it.

**Comment by KnightMiner** [comment]
This report does not help fix anything with no information, at this time you are the only person who is getting a crash, its on you to determine which mods cause it or to give a useful crash report.

Did you check the crash reports folder? Are you sure this is the log from the crash? Typically the proper log is `latest.log`, clear out the logs folder and boot again to get it.

**Comment by ItsJustDance** [comment]
its a new instance, a crash report folder doesn't even exist yet. I had recreated the issue many times, and just put in the prior log rather than the latest. Its probably a problem on my end, you can close the issue if you want.

**Comment by Insane96** [comment]
Do you have (in the main minecraft folder) files called something like `hs_*`?

**Comment by JustWaggs** [comment]
Looks like Mantle requires forge 40.1.9 or above though I don't believe there is a way to download this version of forge. Hope this helps!

[latest.log](https://github.com/SlimeKnights/Mantle/files/8932965/latest.log)
[crash-2022-06-18_01.14.17-fml.txt](https://github.com/SlimeKnights/Mantle/files/8932967/crash-2022-06-18_01.14.17-fml.txt)

**Comment by Insane96** [comment]
The latest version of Forge is 40.1.51

**Comment by JustWaggs** [comment]
yeah I don't know that's just what the log says

**Comment by KnightMiner** [comment]
Yes, the log says your forge is old. You are using 4.1.0 as clearly said by the log. That is too old. 

You can clearly download newer forge than you have. 51 newer versions.

**Comment by JustWaggs** [comment]
sorry I didnt realize that 40.1.9 is an older version than 40.1.51 clearly

**Comment by him126163** [comment]
i have another problems.When i install the mantles,the game will crash.I install the latest forge vision and it also crash
[minecraft-exported-logs-2022-07-01T09-11-39.log](https://github.com/SlimeKnights/Mantle/files/9024333/minecraft-exported-logs-2022-07-01T09-11-39.log)

**Comment by Insane96** [comment]
Remove optifine and try again

**Comment by KnightMiner** [comment]
This is an issue tracker, not a support forum. Please do not post comments here saying "I have another problem", another problem belongs on a separate issue. And please avoid posting saying "I have the same problem" unless you understand both your problem and the problem in this issue well enough to verify they are the same. It is really difficult to attempt to provide support to three users with different problems in one issue, that is why separate issues exist. Not every crash on startup is the same problem.

In other words, unless you are @ItsJustDance (or are 100% certain that your issue is the same based on your knowledge of Minecraft code) do not post your stack trace here. Make a new issue.

**Comment by Cozmo531** [comment]
I was too having this problem, and I have been troubleshooting, it seems that Optifine **is** a mod that can and will cause mantle to crash.

**Comment by KnightMiner** [comment]
I am going to close this for lack of info. Original poster never gave a log and others were just posting unrelated issues.

For anyone finding this in the future, update Forge and don't use broken dumpster fires like Optifine
